boards/FSM/Src/Io/Io_Brake.c

```diff
@@ -1,7 +1,81 @@
 #include "main.h"
 #include "Io_Brake.h"
+#include "Io_Adc.h"
 
+// TODO set these values
+#define BRAKE_PEDAL_MIN_VOLTAGE (0.0f)
+#define BRAKE_PEDAL_MAX_VOLTAGE (1.0f)
+
+#define BRAKE_PRESSURE_OC_THRESHOLD_V (0.33f)
+#define BRAKE_PRESSURE_SC_THRESHOLD_V (3.0f)
+#define BRAKE_PRESSURE_SENSOR_MAX_V (5.0f)
+
+// TODO should we keep this/should this be reimplemented? This needs including "main.h"
 bool Io_Brake_IsActuated(void)
```

Comment:
we should keep this. I would like to see better abstraction, but it will be useful for soft-faulting before bspd fault